### PR TITLE
Show "unsupported" window under wayland

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: config/main_window.cpp [hunk 1]
```diff
@@ -84,6 +84,15 @@ MainWindow::MainWindow(QWidget *parent)
     connect(mActions, &Actions::daemonAppeared,                  this, &MainWindow::daemonAppeared);
     connect(mActions, &Actions::multipleActionsBehaviourChanged, this, &MainWindow::multipleActionsBehaviourChanged);
 
+     // Show info under wayland
+            if (QGuiApplication::platformName() == QLatin1String("wayland"))
```
Comment (Chiitoo) [inline config/main_window.cpp:88]: Is it just me, or is the indentation a bit odd here.

Comment (stefonarch) [reply config/main_window.cpp]: Uhm thanks, hopefully ok now.

Comment (Chiitoo) [reply config/main_window.cpp]: Looks a bit more like the rest fer sure.  : ]

**Comment by tsujan** [comment]
IMO, if this should be done under Wayland, it may be better to do it before making the config window, in `main.cpp`..

**Comment by stefonarch** [comment]
> IMO, if this should be done under Wayland, it may be better to do it before making the config window, in `main.cpp`.

That's true, done.

**Comment by tsujan** [comment]
It looks good. I should also test it later, before giving a "GTM".

**Review by tsujan — APPROVED** [review]
Works as intended.

**Pull request merged**